code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -804,37 +811,31 @@
 	name = "Service"
 	basic_modules = list(
 		/obj/item/assembly/flash/cyborg,
-		//obj/item/reagent_containers/borghypo/borgshaker, //bubber-edit
+		//NOVA EDIT START
+		// /obj/item/reagent_containers/borghypo/borgshaker,
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/alcohol,
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/soda,
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/juice,
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/misc,
 		/obj/item/borg/apparatus/beaker/service,
+		/obj/item/borg/apparatus/beaker, // Allows the pickup of different beakers for easier drink mixing
+		//NOVA EDIT END
 		/obj/item/reagent_containers/cup/beaker/large, //I know a shaker is more appropiate but this is for ease of identification
-		//Skyrat Edit Start: Borg Buff
-		//obj/item/reagent_containers/condiment/enzyme, //edit - Borg shaker has it
-		/obj/item/borg/apparatus/beaker, // SKYRAT EDIT: allows the pickup of different beakers for easier drink mixing
+		//obj/item/reagent_containers/condiment/enzyme, // NOVA EDIT - Borg shaker has it
 		/obj/item/reagent_containers/dropper,
-		//obj/item/reagent_containers/condiment/enzyme, //edit
-		//obj/item/reagent_containers/condiment/enzyme, //bubber-edit
-		//obj/item/reagent_containers/dropper, //bubber-edit
+		/obj/item/reagent_containers/syringe, //NOVA EDIT
 		/obj/item/rsf,
 		/obj/item/storage/bag/tray,
-		/obj/item/storage/bag/tray, // SKYRAT EDIT: Moves the second tray up to be near the default one
+		/obj/item/storage/bag/tray, // NOVA EDIT: Moves the second tray up to be near the default one
+		/obj/item/cooking/cyborg/power, //NOVA EDIT
+		/obj/item/storage/bag/plants/cyborg, //bubber-edit
```

Comment:
Modularity.
Example
```
/obj/item/robot_model/service/Initialize()
	basic_modules += list(/obj/item/storage/bag/plants/cyborg)
	. = ..()
```

Reply:
This file is modified this way on both bubber and skyrat already. Too late to go that route + outside the scope of this PR. We can clean it up later.

Reply:
It'd also prefer it get cleaned up on Nova so that Bubber just has to merge downstream from it.